addToGraph.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Created by dusting on 11/4/15.
+ */
+var _ = require('underscore');
+var internal = exports.internal = {};
+
+exports.addToGraph = function(acc, witnessAccount) {
+    internal.previousEvent = null;
+    internal.acc = acc;
+    _.each(witnessAccount, internal.addEvent);
+    return acc;
+
+};
+
+internal.addEvent = function(event) {
```

Comment:
I don't think you call `internal.addEvent` outside of this module. You should just change this to.

```
//line:10
_.each(witnessAccount, addEvent);

//line:15
function addEvent(event) {
```